### PR TITLE
added nginx alongside qgis server docs

### PR DESCRIPTION
**Pull request by tudorbarascu**

### CODE REVIEW

**Comment by Gustry** [comment]
Thanks for adding some documentations about nginx

**Comment by tudorbarascu** [comment]
@Gustry pleasure. Hope it's useful. Feedback is welcomed.

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 1]
```diff
@@ -863,3 +863,65 @@ This is a list of the variables supported by QGIS server:
   the ID in the :file:`srs.db`. Look at attribute **coord_op_code** of table
   **tbl_datum_transform** in :file:`srs.db` to find the correct entry.
 
+.. index:: nginx
+
+QGIS Server with NGINX
+======================
+
+You can use QGIS Server with `nginx: <http://nginx.org/>`_.
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: why do you add : after nginx label?

Comment (tudorbarascu) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: Typo.. fixed

**Pull request merged**